docker-compose.yml

```diff
@@ -148,24 +147,54 @@ services:
     networks:
       - default
 
-#   storage:
-#     image: ${DOCKER_REPOSITORY}renga-storage:$PLATFORM_VERSION
-#     environment:
-#       AUTHORIZATION_CLIENT_SECRET: $AUTHORIZATION_CLIENT_SECRET
-#       PLAY_APPLICATION_SECRET: $PLAY_APPLICATION_SECRET
-#       RENGA_SERVICES_AUTHORIZATION_URL: ${RENGA_ENDPOINT}/auth/realms/Renga/protocol/openid-connect/auth
-#       RESOURCE_MANAGER_PUBLIC_KEY: $RESOURCE_MANAGER_PUBLIC_KEY
-#       STORAGE_BACKEND_LOCAL_ENABLED: "true"
-#     volumes:
-#     - ./services/storage/data:/data
-#     ports:
-#     - 9000
-#     restart: on-failure
-#     labels:
-#       traefik.enable: "true"
-#       traefik.frontend.rule: PathPrefix:/api/storage
-
+  storage:
+    image: ${DOCKER_REPOSITORY}renga-storage:${PLATFORM_VERSION}
+    environment:
+      AUTHORIZATION_CLIENT_SECRET: $AUTHORIZATION_CLIENT_SECRET
+      PLAY_APPLICATION_SECRET: $PLAY_APPLICATION_SECRET
+      RENGA_SERVICES_AUTHORIZATION_URL: ${RENGA_ENDPOINT}/auth/realms/Renga/protocol/openid-connect/auth
```

Comment:
this might be `http://keycloak.${...}/realms...`

Reply:
This is consistent with line 63. Further changes to using domain name could be done in a separate PR.

Reply:
`docker-compose` has changed in the meantime. Unless we choose to revert #133 with #138 this is inconsistent. In any case, you can use `${KEYCLOAK_URL}/auth/...` here for the time being and I think you're ok with both options.

Reply:
Changed it. Gitlab runner will probably also need this change in its configuration. https://github.com/SwissDataScienceCenter/renga/pull/128/files#diff-4e5e90c6228fd48698d074241c2ba760R148

Reply:
ah yes, absolutely, good catch.